src/Nethermind/Nethermind.Blockchain/Producers/DevBlockProducer.cs

```diff
@@ -37,12 +37,13 @@ public class DevBlockProducer : BaseBlockProducer
             ITxSource txSource,
             IBlockchainProcessor processor,
             IStateProvider stateProvider,
+            ISigner signer,
             IBlockTree blockTree,
             IBlockProcessingQueue blockProcessingQueue,
             ITxPool txPool,
             ITimestamper timestamper,
             ILogManager logManager) 
-            : base(txSource, processor, NethDevSealEngine.Instance, blockTree, blockProcessingQueue, stateProvider, timestamper, logManager)
+            : base(txSource, processor, NethDevSealEngine.Instance, blockTree, blockProcessingQueue, stateProvider, signer, timestamper, logManager)
```

Comment:
Dev block producer is not signing - and it has to have a signer passed -> a follow up of the previous comments